v2/concordium/service.proto

```diff
@@ -139,4 +139,12 @@ service Queries {
   // for API consistency.
   // The stream will end when all the non-finalized transaction hashes have been returned.
   rpc GetAccountNonFinalizedTransactions (AccountAddress) returns (stream TransactionHash);
+
+   // Start dumping packages into the specified file.
+  // Only enabled if the node was built with the `network_dump` feature.
+  rpc DumpStart(DumpRequest) returns (BoolResponse) {}
```

Comment:
I'd recommend using `bool` here if we need a boolean.

But looking at the implementation we don't actually return anything useful here. We sometimes return `false` when there is an error. That would be better to handle by returning a proper error, and just return `Empty` here.

And update documentation.